updaters.py

```diff
@@ -0,0 +1,74 @@
+""" Updater for regular items. """
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractUpdater(ABCMeta):
```

Comment:
Abstract classes are overcomplicated and hugely misstaken. There is no need really to create a metaclass with an interface for that. All of it could be a simple functions - whenever you see that you have a class and there is only one method and it's `@staticmethod` you should think twice if you ever need one. In most cases you dont.

Still, great code, you've nailed the task!